test/relayer_test.go

```diff
@@ -13,8 +13,8 @@ import (
 func getTestChainFactory() ibctest.ChainFactory {
 	return ibctest.NewBuiltinChainFactory(
 		[]ibctest.BuiltinChainFactoryEntry{
-			{Name: "gaia", Version: "v6.0.4", ChainID: "cosmoshub-1004", NumValidators: 1, NumFullNodes: 1},
-			{Name: "osmosis", Version: "v7.0.4", ChainID: "osmosis-1001", NumValidators: 1, NumFullNodes: 1},
+			{Name: "gaia", Version: "latest", ChainID: "cosmoshub-1004", NumValidators: 1, NumFullNodes: 1},
```

Comment:
I like `latest` for the example matrix file and for the default behavior of the `ibctest` binary we ship, but I don't like the idea that the tests for the same commit of `ibc-test-framework` could change behavior based on an external docker image changing.

The tradeoff, of course, is that we have to update ibc-test-framework whenever those images change. That can be mostly automated with a little effort.

Reply:
+1

Reply:
works for me, then it's just one place to maintain instead of 3